src/libs/actions/BankAccounts.js

```diff
@@ -842,6 +843,10 @@ function setWorkspaceIDForReimbursementAccount(workspaceID) {
     Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT_WORKSPACE_ID, workspaceID);
 }
 
+function updateReimbursementAccountDraft(bankAccountData) {
+    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT, {...bankAccountData});
```

Comment:
Can you also confirm that the changes surrounding `oldAchData` https://github.com/Expensify/App/pull/4477/files#diff-448c3537222b5e687caa8f06585ace6feae03d751f243e1590383fb6a33693e8R331 are still necessary? Just to double-check we don't have duplicated unnecessary code there

Reply:
Nice one! They are indeed not necessary anymore and I have removed them. I also ran the test steps in https://github.com/Expensify/App/pull/4477 and it worked well 😄

Reply:
Is there some reason we should be using spread here instead of setting the object without doing that?

Reply:
I used the spread so we could store all the key-value pairs at the top level, instead of inside a nested `bankAccountData` object, like so:

```
{
   routingNumber: '12345',
   accountNumber: '12345',
}
```

vs 

```
{
   bankAccountData: {
       routingNumber: '12345',
       accountNumber: '12345',
   },
}
```

Reply:
Ok, so what is the difference between 

```js
Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT, {...bankAccountData});
```

and 

```js
Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT, bankAccountData);
```

Reply:
🤦  Thanks!